easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-4.5.2-foss-2016a-METIS-5.1.0.eb

```diff
@@ -0,0 +1,27 @@
+name = 'SuiteSparse'
+version = '4.5.2'
+
+homepage = 'http://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'opt': True, 'unroll': True, 'pic': True}
+
+source_urls = ['http://faculty.cse.tamu.edu/davis/SuiteSparse/']
+sources = [SOURCE_TAR_GZ]
+
+metis = 'METIS'
+metis_ver = '5.1.0'
+versionsuffix = '-%s-%s' % (metis, metis_ver)
+dependencies = [(metis, metis_ver)]
+
+preconfigopts = 'MY_METIS_LIB="$EBROOTMETIS/lib/libmetis.so" ' 
+preconfigopts += 'LAPACK="$LIBLAPACK" '
```

Comment:
ok, please remove this SuiteSparse easyconfig too then, since it'll need some work (cfr. https://github.com/hpcugent/easybuild-easyconfigs/pull/3050)